concourse/scripts/docker-compose-fauna.yml

```diff
@@ -2,7 +2,7 @@ version: "3.3"
 
 services:
   faunadb:
-    image: fauna/faunadb:4.28.0
+    image: fauna/faunadb:5.10.0
```

Comment:
maybe we should juset pin to latest.

Reply:
I don't think I have any objections to that.